examples/url/src/lib.rs

```diff
@@ -75,9 +75,9 @@ fn view(model: &Model) -> Node<Msg> {
                 ev(Ev::Click, |_| {
                     Url::new()
                         .set_path(&["ui", "a", "b", "c"])
-                        .set_search(UrlSearch::new(vec![
+                        .set_search(vec![
                             ("x", vec!["1"])
-                        ]))
+                        ].into_iter().collect())
```

Comment:
I don't know if it's better or worse: two calls + hidden type VS explicit type `UrlSearch` that needs to be imported.
Do you think we can write only `.set_search(vec![("x", vec!["1"])])` or `set_search(UrlSearch::new..)` and don't complicate `set_search` parameter too much?